gcloud_requests/proxy.py

```diff
@@ -76,31 +64,30 @@ def __init__(self, credentials=None, logger=None):
     def __del__(self):
         _credentials_watcher.unwatch(self.credentials)
 
-    def request(self, uri, method="GET", body=None, headers=None, redirections=5, connection_type=None, retries=0, refresh_attempts=0):   # noqa
+    def request(self, method, url, data=None, headers=None, retries=0, refresh_attempts=0, **kwargs):
         session = self._get_session()
         headers = headers.copy() if headers is not None else {}
         auth_request = AuthRequest(session=session)
         retry_auth = partial(
             self.request,
-            uri=uri, method=method,
-            body=body, headers=headers,
-            redirections=redirections,
-            connection_type=connection_type,
+            url=url, method=method,
+            data=data, headers=headers,
             refresh_attempts=refresh_attempts + 1,
             retries=0,  # Retries intentionally get reset to 0.
+            **kwargs
```

Comment:
Was there a particular reason to pass through **kwargs? (besides the fact that it should have been done in the first place?) :trollface:

Reply:
It's part of the API signature now, so if anything google cloud's calling through that `request()` method could potentially be passing down a lot more arguments than what we define here.